xoa_driver/functions/anlt.py

```diff
@@ -106,6 +142,32 @@ async def port_force_reserve(port: GenericAnyPort) -> None:
     return None
```

Comment:
Why this function need to have over actions of creating the list, adding new values to it and after unpacking it.
Can be done simpler.
``` python
r = await port.reservation.get()
if r.status == ReservedStatus.RESERVED_BY_OTHER:
    await apply(
        port.reservation.set_relinquish(),
        port.reservation.set_reserve(),
   )
elif r.status == ReservedStatus.RELEASED:
     await port.reservation.set_reserve()
```